tuxemon/states/pc_locker/__init__.py

```diff
@@ -257,11 +256,7 @@ def __init__(self, box_name: str) -> None:
 
         # menu
         theme.title = True
-        theme.title_background_color = (197, 232, 234)
-        theme.title_font_size = round(0.025 * width)
-        theme.title_font_color = (10, 10, 10)
-        theme.title_close_button = False
-        theme.title_bar_style = MENUBAR_STYLE_ADAPTIVE
+        theme.title_font_size = 30
```

Comment:
notice that `theme.title_font_size = round(0.025 * width)`.  width is based off screen size, which means that it is scaled to the pixel size of the window.  by setting `30` here, the fotn will only be the correct size at whatever resolution you are using now.  we support multiple resolutions, so this value should be scaled to the screen somehow.  you can use `width`, or prepare.SCALE so the font is always the right size.